src/styles/ua.scss

```diff
@@ -0,0 +1,169 @@
+.btn {
+  display: inline-flex;
+  min-height: 60px;
+  min-width: 280px;
+  padding: 5px 15px;
+
+  justify-content: center;
+  align-items: center;
+
+  background-color: $colorMain;
+  color: $colorGrey2;
+  font-family: $fontSharp;
+
+  text-transform: uppercase;
+  letter-spacing: 0.15em;
+
+  border: 1px solid transparent;
+
+  @media (max-width: 480px){
+      width: 100%;
+  }
+
+  @media (min-width: 769px) {
+    animation-name: wow;
+    animation-duration: 10s;
+    animation-timing-function: ease-in-out;
+    animation-iteration-count: infinite;
+    animation-direction: alternate;
+
+    @keyframes wow {
```

Comment:
wow isn’t the best name for animation. Naming the animation describes what this animation is doing.